subcommands/devices/list_denied.go

```diff
@@ -0,0 +1,76 @@
+package devices
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/cheynewallace/tabby"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	paginationLimits = []int{10, 20, 30, 40, 50, 100, 200, 500, 1000}
```

Comment:
I think it's worth extracting this as a constant... it's the same for both device list and denied device list.
Maybe, even extract into common (if can be done easily, not sure about this).